test/chpldoc/compflags/version/projectVersion.docs.cleanfiles

```diff
@@ -0,0 +1 @@
+docs
```

Comment:
Looks like this file isn't triggering because it has an extra s in the "doc" portion.

Reply:
That causes the first five checks to fail

Reply:
Ohh! My bad. I did run the test only once and it was passing. The second time it is faling.
I'll fix it rn.